pkg/installpackage/download.go

```diff
@@ -24,9 +24,17 @@ func (is *Installer) downloadWithRetry(ctx context.Context, logE *logrus.Entry,
 		logE.Debug("check if the package is already installed")
 		finfo, err := is.fs.Stat(param.Dest)
 		if err != nil { //nolint:nestif
-			// file doesn't exist
+			// file doesn't exist or download fails
 			if err := is.download(ctx, logE, param); err != nil {
-				if strings.Contains(err.Error(), "file already exists") {
+				errorSubstrings := []string{"file already exists", "download and unarchive the package"}
```

Comment:
The log `download and unarchive the package` is outputted here.

```console
$ git grep "download and unarchive the package"
pkg/installpackage/download.go: logE.Info("download and unarchive the package")
```

https://github.com/aquaproj/aqua/blob/d553700d5cd06bec75c0ded3bd41346a08bf4910/pkg/installpackage/download.go#L75

Seems like the message isn't included in the error object.

Reply:
According to the code, I don't think this change has meaning because `err` never include the message `download and unarchive the package`.